File: Input.js
const readline = require('readline');

const rl = readline.createInterface({
    input: process.stdin,
    output:process.stdout
});

rl.question('Massukan nama: ', (nama) => {
    rl.question('Masukan umur: ', (umur) => {
        console.log(`Hola ${nama}, umur kamu ${umur} tahun.`);
        rl.close();
    });
});

File: Input.cpp
#include <iostream>
#include <string>
using namespace std;

int main() {
    string nama;
    int umur;

    cout << "Masukkan nama lengkap: ";
    getline(cin, nama);

    cout <<"Masukkan umur: ";
    cin >> umur;

    cout << "Halo: " << nama << ", umur kamu" << umur << " tahun." << endl;

    return 0;
}